src/KDGui/platform/linux/wayland/linux_wayland_platform_window.cpp

```diff
@@ -228,15 +230,15 @@ void LinuxWaylandPlatformWindow::handleTextInput(const std::string &str)
     CoreApplication::instance()->sendEvent(m_window, &ev);
 }
 
-void LinuxWaylandPlatformWindow::enter(wl_surface *surface, wl_output *output)
+void LinuxWaylandPlatformWindow::enter(wl_surface * /*surface*/, wl_output *output)
```

Comment:
why not use KD_UNUSED?

Reply:
Not really a strong preference. I just thought that this way we won't introduce additional lines so just a little less visual clutter.

Reply:
Done